src/java/com/zimbra/qa/selenium/projects/ajax/tests/calendar/meetings/organizer/singleday/create/CreateMeeting.java

```diff
@@ -187,4 +188,52 @@ public void CreateMeeting_03() throws HarnessException {
 		invite = MailItem.importFromSOAP(ZimbraAccount.AccountB(), "subject:("+ appt.getSubject() +")");
 		ZAssert.assertNotNull(invite, "Verify the invite is received");
 	}
+
+
+	@Test (description = "Create basic meeting invite by selecting date from date picker and time from dropdown",
+			groups = { "smoke", "L0",})
+
+	public void CreateMeeting_04() throws HarnessException {
+
+		// Create appointment data
+		AppointmentItem appt = new AppointmentItem();
+
+		String apptSubject, apptAttendee1, apptContent;
+		Calendar now = Calendar.getInstance();
+		apptSubject = ConfigProperties.getUniqueString();
+		apptAttendee1 = ZimbraAccount.AccountA().EmailAddress;
+		apptContent = ConfigProperties.getUniqueString();
+
+		appt.setSubject(apptSubject);
+		appt.setAttendees(apptAttendee1);
+
+		FormApptNew apptForm = (FormApptNew) app.zPageCalendar.zToolbarPressButton(Button.B_NEW);
+		apptForm.zFillField(Field.Subject,apptSubject);
+		apptForm.zFillField(Field.Attendees, apptAttendee1);
+		apptForm.zFillField(Field.Body, apptContent);
+
+		//if it is last day of month than set appointment for same day
```

Comment:
Minor to rename meridiem to something meaningful variable name if possible? And regarding formatting to use equal spaces after = and {. For e.g.

if (now.get(Calendar.DAY_OF_MONTH) == now.getActualMaximum(Calendar.DAY_OF_MONTH)) {
	now.set(Calendar.DAY_OF_MONTH, now.get(Calendar.DAY_OF_MONTH) - 1 );
}

String meridiem = null;
if (now.get(Calendar.AM_PM) == 0) {
	meridiem = "AM";
} else {
	meridiem = "PM";
}